src/components/v5/shared/UserAvatarPopover/UserAvatarPopover.tsx

```diff
@@ -1,16 +1,47 @@
 import React, { FC } from 'react';
 
+import clsx from 'clsx';
 import { UserAvatarPopoverProps } from './types';
 import UserAvatar from '~v5/shared/UserAvatar';
 import UserPopover from '../UserPopover';
+import { useColonyContext } from '~hooks';
+import { splitWalletAddress } from '~utils/splitWalletAddress';
+import { useGetColonyContributorQuery } from '~gql';
+import { getColonyContributorId } from '~utils/members';
+import { ContributorTypeFilter } from '~v5/common/TableFiltering/types';
 
 const displayName = 'v5.UserAvatarPopover';
 
 const UserAvatarPopover: FC<UserAvatarPopoverProps> = ({ size, ...props }) => {
-  const { user, walletAddress, userStatus, isContributorsList } = props;
+  const { walletAddress, isContributorsList } = props;
+  const { colony } = useColonyContext();
+  const { colonyAddress = '' } = colony || {};
+  const { data } = useGetColonyContributorQuery({
+    variables: {
+      id: getColonyContributorId(colonyAddress, walletAddress),
+      colonyAddress,
+    },
+  });
+
+  const contributor = data?.getColonyContributor;
+  const { user } = contributor ?? {};
+  const { bio, displayName: userDisplayName } = user?.profile || {};
+
+  const splitAddress = splitWalletAddress(user?.walletAddress || '');
+  const userStatus = (contributor?.type?.toLowerCase() ??
+    null) as ContributorTypeFilter | null;
 
   return (
-    <UserPopover {...props}>
+    <UserPopover
+      userName={userDisplayName ?? splitAddress}
+      aboutDescription={bio || ''}
```

Comment:
This prop seems unnecessary. In all instances of `UserPopover` it is set to user's profile bio, so it can be removed for now

Reply:
@jakubcolony I have checked that, and in that case we have to pass `bio` otherwise on Verified page in user popover description won't display

Reply:
But the `bio` comes from the user's profile object, right? And `UserPopover` already fetches a user object so could it not just get the `bio` from the query data, instead of being passed a prop?

Reply:
fixed